deploy/quickstart/quick_start.py

```diff
@@ -0,0 +1,245 @@
+        replace_in_file(docker_compose, './nginx.conf', nginx)
+        replace_in_file(docker_compose, '../capabilities-config.json',
+                        capabilities_config)
+        replace_in_file(docker_compose, '../api-config.json', api_config)
+        replace_in_file(docker_compose, 'USE_CAAS=True', 'USE_CAAS=False')
+
+        # This will obviously fail if the capabilities config gets any more
+        # than one default-required field like this:
+        replace_in_file(capabilities_config, '\'isRequired\': true',
+                        '\'isRequired\': false')
+
+        print('To start, run:')
+        print('> ' + start_script_file)
+
+    else:
+        print(
+            'Sorry, not yet implemented! Please raise this issue at https://github.com/DataBiosphere/job-manager/issues'
+        )
+        exit(1)
+
+
+def quick_start_dsub(version, install_dir, bin_dir, config_dir):
+    print(
+        'Sorry, not yet implemented! Please raise this issue at https://github.com/DataBiosphere/job-manager/issues'
+    )
+    exit(1)
+
+
+def download_file(version, file, to_path):
+    abs_to_path = path.abspath(to_path)
+    from_url = 'https://raw.githubusercontent.com/DataBiosphere/job-manager/{0}/{1}'.format(
+        version, file)
+    urllib.request.urlretrieve(from_url, abs_to_path)
+    return abs_to_path
+
+
+def replace_in_file(file, to_replace, replace_with):
+    with open(file, 'rt') as fin:
+        temp = 'temp_out.txt'
+        with open('temp_out.txt', 'wt') as fout:
+            for line in fin:
+                fout.write(line.replace(to_replace, replace_with))
+        call(['mv', '-f', temp, file])
+
+
+def write_string(file, lines):
+    with open(file, 'wt') as fout:
+        for line in lines:
+            fout.writelines(line + '\n')
+
+
+def pull_docker(name, version):
+    docker_image = name + ':' + version
+    print('Pulling docker image {0}'.format(docker_image))
+    call(['docker', 'pull', docker_image], stdout=subprocess.DEVNULL)
+    return docker_image
+
+
+def request_input_from_list(message, options, default):
+    valid = False
+    provided = default
+    if default not in options:
+        all_options = options.copy()
+        all_options = all_options + default
+    else:
+        all_options = options
+
+    printed_options_list = '({0})'.format('/'.join(
+        str(e) for e in all_options))
+
+    while not valid:
+        provided = input('>> {0} {1} [ {2} ] >'.format(
+            message, printed_options_list, default)) or default
+        if provided in all_options:
+            valid = True
+        else:
+            print('Invalid option. Expected one of: {0} but got: {1}'.format(
+                printed_options_list, provided))
+            valid = False
+    print('Using: ' + provided)
+    print()
+    return provided
+
+
+def make_or_replace_or_reuse_directory(path_to_make, exit_if_invalid):
+    valid = False
+    time_to_exit = False
+    while not time_to_exit:
```

Comment:
nit: It would be great to have some separators before each call of this function, since this function will be called more than once, I was confused(thinking it's an infinite loop) before I looked into the actual msg. A possible approach is to use a long line based on the console width: 
```
import os
print('=' * int(os.popen('stty size', 'r').read().split()[1]))
```